workgroup/overrides.json

```diff
@@ -1,38 +1,38 @@
 {
-  "CREATE": {
-    "/Tags": [
-      {
-        "Key": "tagKey1",
-        "Value": "tagValue1"
-      },
-      {
-        "Key": "tagKey2",
-        "Value": "tagValue2"
-      }
-    ],
-    "/WorkGroupConfiguration/ResultConfiguration/OutputLocation": "s3://bucket",
-    "/WorkGroupConfiguration/EngineVersion/EffectiveEngineVersion": "Athena engine version 2",
-    "/WorkGroupConfiguration/EngineVersion/SelectedEngineVersion": "AUTO",
-    "/WorkGroupConfigurationUpdates/ResultConfigurationUpdates/OutputLocation": "s3://bucket",
-    "/WorkGroupConfigurationUpdates/EngineVersion/EffectiveEngineVersion": "Athena engine version 2",
-    "/WorkGroupConfigurationUpdates/EngineVersion/SelectedEngineVersion": "AUTO"
-  },
-  "UPDATE": {
-    "/Tags": [
-      {
-        "Key": "tagKey1",
-        "Value": "tagValue1"
-      },
-      {
-        "Key": "tagKey2",
-        "Value": "tagValue2"
-      }
-    ],
-    "/WorkGroupConfiguration/ResultConfiguration/OutputLocation": "s3://bucket",
-    "/WorkGroupConfiguration/EngineVersion/EffectiveEngineVersion": "Athena engine version 2",
-    "/WorkGroupConfiguration/EngineVersion/SelectedEngineVersion": "AUTO",
-    "/WorkGroupConfigurationUpdates/ResultConfigurationUpdates/OutputLocation": "s3://bucket",
-    "/WorkGroupConfigurationUpdates/EngineVersion/EffectiveEngineVersion": "Athena engine version 2",
-    "/WorkGroupConfigurationUpdates/EngineVersion/SelectedEngineVersion": "AUTO"
-  }
-}
\ No newline at end of file
+    "CREATE": {
+        "/Tags": [
+            {
+                "Key": "tagKey1",
+                "Value": "tagValue1"
+            },
+            {
+                "Key": "tagKey2",
+                "Value": "tagValue2"
+            }
+        ],
+        "/WorkGroupConfiguration/ResultConfiguration/OutputLocation": "s3://bucket",
+        "/WorkGroupConfiguration/EngineVersion/EffectiveEngineVersion": "Athena engine version 2",
+        "/WorkGroupConfiguration/EngineVersion/SelectedEngineVersion": "AUTO",
+        "/WorkGroupConfigurationUpdates/ResultConfigurationUpdates/OutputLocation": "s3://bucket",
+        "/WorkGroupConfigurationUpdates/EngineVersion/EffectiveEngineVersion": "Athena engine version 2",
+        "/WorkGroupConfigurationUpdates/EngineVersion/SelectedEngineVersion": "AUTO"
+    },
+    "UPDATE": {
+        "/Tags": [
+            {
+                "Key": "tagKey1",
+                "Value": "tagValue1"
+            },
+            {
+                "Key": "tagKey2",
+                "Value": "tagValue2"
+            }
+        ],
+        "/WorkGroupConfiguration/ResultConfiguration/OutputLocation": "s3://bucket",
+        "/WorkGroupConfiguration/EngineVersion/EffectiveEngineVersion": "Athena engine version 2",
+        "/WorkGroupConfiguration/EngineVersion/SelectedEngineVersion": "AUTO",
```

Comment:
[minor] should we have some updated config in the update overrides so we can test that it's actually updated?

Reply:
Would the contract tests be able to check that? It was my understanding that these were checking basic functionality such as error handling and validating input, whereas making sure update is actually updating would be more of a responsibility for integration tests

Reply:
I'm not 100% sure if the contract tests check it, but you're right, we can leave it for the integ test.